spec/configuration_spec.rb

```diff
@@ -36,6 +36,13 @@
     end
   end
 
+  describe '#exclude' do
+    it 'should parse the exclude list correctly' do
```

Comment:
Wouldn't it be nice to have a test for the actual rejection from the dependencies?